README.md

```diff
@@ -0,0 +1,168 @@
+# ROCM-CORE Introduction
+
+ROCM-CORE is a package which can be used to get ROCm release version, get ROCm install path information etc.
+It is also important to note that ROCM-CORE takes the role as a base component on which all of ROCm can depend,
+to make it easy to remove all of ROCm with a package manager.
+
+getROCmVersion function provides the ROCm version. 
+
+It also provides an example Lmod modules files for the ROCm release.
+
+Lmod module files can be loaded with the following commands.
+``` shell
+module load rocm/x.y or 
+module load rocm
+```
+
+You can find sources and binaries in our [GitHub repository](https://github.com/ROCm/rocm-core).
+
+> [!NOTE]
+> As with all ROCm projects, the documentation is open source. For more information, see [Contribute to ROCm documentation](https://rocm.docs.amd.com/en/latest/contribute/contributing.html).
+
+## Installing ROCM-CORE
+
+Before we proceed with how to install, take a look on references given below  to understand System requirements, ROCm Installation prerequisites, ROCm package repository set up etc.
+
+### Prerequisite References
+
+* Refer [System Requirements](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.2.2/reference/system-requirements.html#supported-gpus)
+* Refer [ROCm installation for Linux](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.2.2/index.html)
+* A [ROCm](https://rocm.docs.amd.com/)-enabled platform
+* To be noted that ROCM-CORE library primarily depends on having the C library available for the respective platform.
+
+### Installing with pre-built packages
+
+You can install ROCM-CORE on Ubuntu using
+
+```shell
+apt-get install rocm-core
+```
+
+## Building ROCM-CORE from source
+
+You can build ROCM-CORE form source
+
+First, get the sources from repository.
+
+```shell
+  git clone https://github.com/ROCm/rocm-core.git
+```
+
+Go to Root Directory, create a build directory:
+
+```shell
+  cd rocm-core; mkdir build; cd build
+```
+
+Next, configure CMake. Invoke cmake with the following variables define as deemed fit
+
+```shell
+cmake -S $PWD/../ -B . \
+    -DCMAKE_VERBOSE_MAKEFILE=1 \
+    -DCMAKE_INSTALL_PREFIX=./ \
+    -DROCM_VERSION="6.4.0" \
+    ..
+```
+
+>[!NOTE]
+>When specifying the path for the `CMAKE_PREFIX_PATH` variable, **do not** use the tilde (`~`)
+>shorthand to represent the home directory.
+### Setting up install locations
+By default, the install location is set to `/opt/rocm`. You can change this using
+`CMAKE_INSTALL_PREFIX`:
+```shell
+cmake -DCMAKE_INSTALL_PREFIX=<rocm-core-install-path> ..
+```
+where rocm-core-install-path is "./" in the cmake configure command shared above.
+
+### Install
+
+Once cmake configuration successfully completed, from the same build directory run build, install commands
```

Comment:
I would prefer
same build directory run the default (build and then the install targets

Reply:
Updated comments in the latest version